research-software-directory/RSD-as-a-service
# feat: edit software and project mention pages

# Edit software and project mentions pages

Closes #1121
Closes #1076
Closes #1115 

Changes proposed in this pull request:
* One menu options "Mentions" instead of two (mentions and reference papers)
* Three tabs on the page: reference papers, citations, related output    
* This PR includes both software and project mentions!

How to test:
* `make start` to build and generate test data
* login as rsd_admin, create new software or edit existing
* add reference papers:
  * doi:10.1186/s13321-017-0220-4
  * 10.1186/1758-2946-6-3
  * https://doi.org/10.1021/ja026939x
* add, import, create related output
   * for import use the list [from this issue](https://github.com/research-software-directory/RSD-as-a-service/pull/808)
* validate citations are scraped (ensure scrapers are running in dev)
* create new project or edit existing project
* add output and impact items manually
* observe the scrapers generating citations based on the items added to output
* test adding manual items and import

## Edit software mentions
![image](https://github.com/research-software-directory/RSD-as-a-service/assets/9204081/6c244947-d32d-44ec-bed3-a99a71b054d2)

## Edit project mentions
![image](https://github.com/research-software-directory/RSD-as-a-service/assets/9204081/a872d615-4da8-404b-8cc3-b8fd76bdf9d5)

PR Checklist:
